build.gradle.kts

```diff
@@ -8,14 +8,15 @@ version = "0.0.1"
 
 repositories {
     mavenCentral()
-    maven("https://jitpack.io")
+    maven("https://jitpack.io/")
```

Comment:
Remove jitpack

Reply:
MCProtocolLib has a dependency requiring jitpack, even though itself not being on jitpack iirc

Reply:
That's reasonable then